### PR TITLE
Bumped artifact-engine dependency in the common-npm-packages/az-blobstorage-provider-v2 task

### PR DESCRIPTION
**Pull request by sbalia**
**Task name**: common-npm-packages/az-blobstorage-provider-v2

**Description**:
Bumped artifact-engine to remove set-value dependency.

This affects the package set-value before 2.0.1, and starting with 3.0.0 but prior to 4.0.1. A type confusion vulnerability can lead to a bypass of CVE-2019-10747 when the user-provided keys used in the path parameter are arrays.

**Documentation changes required**: N

**Added unit tests**: Y

**Attached related issue**: Y
https://github.com/advisories/GHSA-4jqc-8m5r-9rpr

**Checklist**:

- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [X] Checked that applied changes work as expected (smoke tested in the local pipeline using JavaToolInstallerV0 task)

### CODE REVIEW

**Comment by AndreyIvanov42** [comment]
It looks like this [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/16322) isn't actual at the moment

File: package.json [hunk 1]
```diff
@@ -35,8 +35,6 @@
     "azure-devops-node-api": "^10.2.2",
     "js-yaml": "^3.13.1",
     "minimatch": "3.0.2",
-    "minimist": "^1.2.5",
```
Comment (AndreyIvanov42) [inline package.json:38]: NIT: these changes need to be added with a separate PR that has already been created in several quantities

**Review by AndreyIvanov42 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sbalia) [reply package.json:38]: So, shall I remove it from this pr?

**Comment by sbalia** [comment]
> It looks like this [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/16322) isn't actual at the moment

Well, how can we determine which PR should be left and which should be closed?

**Comment by AndreyIvanov42** [comment]
> > It looks like this [PR](https://github.com/microsoft/azure-pipelines-tasks/pull/16322) isn't actual at the moment
> 
> Well, how can we determine which PR should be left and which should be closed?

It's better to leave this PR, here's a more actual package-lock.json

File: package.json (hunk 1 above)
Comment (AndreyIvanov42) [reply package.json:38]: Let's remove

Comment (sbalia) [reply package.json]: removed

**Pull request merged**